src/pages/deploy/deployments.md

```diff
@@ -104,29 +118,30 @@ width={1518} height={502} quality={80} />
 To perform a rollback, click the three dots at the end of a previous deployment,
 you will then be asked to confirm your rollback.
 
-### Logs
+### Redeploy
 
-Railway allows users to see running logs of your application to help with
-monitoring. Railway displays the last 10,000 lines of logs available for a
-deployment.
+<PriorityBoardingBanner />
 
-We maintain logs for inactive deployments as well as active. Under the logs pane, you can search within your logs for certain keywords.
+A successful, failed, or crashed deployment can be re-deployed. This will create
+an new deployment with the exact same code and build/deploy configuration.
 
 <Image
-src="https://res.cloudinary.com/railway/image/upload/v1645149733/docs/logs_seqcps.png"
-alt="Animation of Log Filtering"
+src="https://res.cloudinary.com/railway/image/upload/v1666380373/docs/redeploy_ghinkb.png"
+alt="Screenshot of Redeploy Menu"
 layout="responsive"
-width={1512} height={1254} quality={80} />
+width={888} height={493} quality={100} />
```

Comment:
I was under the impression that lowering `quality` would make the image load faster? Am I mistaken?

Reply:
It should yes. But I realized that this image is already about half the size as the other images so I up'ed the quality to hopefully be comparable